docs/src/getting-started/basics.md

```diff
@@ -171,6 +171,19 @@ Where `value` will hold the value returned by its respective contract method, re
 
 And lastly, `logs` will hold all logs that happened within that specific contract call.
 
+In order to log out `receipts` values during testing we need to do the following:
+
+1. You have to initialize `tracing_subscriber` in `harness.rs`
```

Comment:
This instruction is redundant because you already wrote the code inside `harness.rs`. The formulation might confuse a user, you should probably just keep `2.`. If you want to provide general instructions on how to set up the tests, you can say something along the lines of `
see init_tracing function inside packages/fuels-abigen-macro/harness.rs`



This will help avoid duplication and updates overhead.